### PR TITLE
Upgrade to OCamlformat 0.27.0

### PR DESCRIPTION
**Pull request by Julow**
The aim of this preview is to gather feedback.

Changelog can be found here: https://github.com/ocaml-ppx/ocamlformat/blob/main/CHANGES.md

### CODE REVIEW

File: src/context_free.ml [hunk 1]
```diff
@@ -653,17 +653,17 @@ class map_top_down ?(expect_mismatch_handler = Expect_mismatch_handler.nop)
...
-           Attribute.remove_seen_res Module_expr
-             [ T Ast_traverse.enter_module ]
-             x
-           |> of_result ~default:x
-           >>| fun x ->
-           (Expansion_context.Base.enter_module ~loc txt base_ctxt, x))
+      ( (* Make sure code-path attribute is applied before expanding. *)
+        Attribute.get_res Ast_traverse.enter_module x |> of_result ~default:None
+      >>= function
+        | None -> return (base_ctxt, x)
```
Comment (NathanReb) [inline src/context_free.ml:659]: Isn't the alignment of `>>= function` a little odd here?

**Review by NathanReb — COMMENTED** [review]
I like the new formatting much better ! Awesome work!

File: src/context_free.ml (hunk 1 above)
Comment (NathanReb) [reply src/context_free.ml:659]: Anyway, nothing critical and I'm happy to merge those changes once 0.27.0 releases!

**Comment by Julow** [comment]
OCamlformat 0.27.0 as been released :tada: This is ready to be merged.

**Comment by NathanReb** [comment]
Thanks @Julow ! I'll merge the urgent PRs and will rebase this!

**Comment by Julow** [comment]
Thanks for looking at the diffs :) I'll close this PR as it's not easily rebased and it's better if the upgrade is done by a maintainer.

**Pull request closed without merging**